multiplexing_server.py

```diff
@@ -0,0 +1,175 @@
+import socket
+import time
+import select
+
+# global variables
+MAX_MSG_LENGTH = 1024
+SERVER_PORT = 5555
+SERVER_IP = "0.0.0.0"
+MANAGERS_ARRAY = ["@ariel", "@shua"]
```

Comment:
the requirements are that people can't set their username with '@',
and when admins send message a '@' will be appended to the start of their name